src/Neo/Ledger/MemoryPool.cs

```diff
@@ -291,6 +296,11 @@ internal bool CanTransactionFitInPool(Transaction tx)
 
         internal VerifyResult TryAdd(Transaction tx, DataCache snapshot)
         {
+            if (_throttler != null && !_throttler.ShouldAcceptTransaction(tx))
+            {
+                return VerifyResult.OutOfMemory;
```

Comment:
OutOf....
This is not Memory itself.

Reply:
The error should be like this. There is also `PolicyFail`, but it's more about the Policy contract (as per NEP-23).